blink.js

```diff
@@ -1 +1,9 @@
 // YOUR CODE GOES HERE
+$.fn.blink = function(seconds) {
+	var $this = this
```

Comment:
Also, I would probably go with `$el` over `$this` so it's more obvious to the reader what it corresponds to.